### PR TITLE
Fix violations of and reenable rubocop Lint/ShadowedArgument.

### PR DESCRIPTION
**Pull request by asherkach**

### CODE REVIEW

**Comment by asherkach** [comment]
CC @islemaster for CircleCI run and review.

File: shared/middleware/shared_resources.rb [hunk 1]
```diff
@@ -64,7 +64,7 @@ def cache(type)
   end
 
   # JavaScripts
-  get "/shared/js/*" do |path|
+  get "/shared/js/*" do |_path|
```
Comment (asherkach) [inline shared/middleware/shared_resources.rb:67]: Is `path` intentionally being shadowed? Not being familiar, I don't know if my fix is correct or masking a bug.

**Comment by islemaster** [comment]
Circle run started.

File: shared/middleware/shared_resources.rb (hunk 1 above)
Comment (islemaster) [reply shared/middleware/shared_resources.rb:67]: Good question.  I suspect your fix is correct.

**Pull request merged**